index.html

```diff
@@ -8,22 +8,30 @@
     <link rel="stylesheet" href="css/styles.css" type="text/css" />
   </head>
   <body>
-    <h1>Learn to code by watching others</h1>
 
-    <p>See how experienced developers solve problems in real-time. Watching scripted tutorials is great, 
-    but understanding how developers think is invaluable. </p>
-  
-    <div>Try it free 7 days then $20/mo. thereafter</div>
-    <div>
-      First Name 
-      Last Name 
-      Email Address
-      Password
-  
-      <div>Claim your free trial </div>
+    <div class="text-blurb">
+      <h1>Learn to code by watching others</h1>
+      <p>See how experienced developers solve problems in real-time. Watching scripted tutorials is great, 
+      but understanding how developers think is invaluable. </p>
+    </div>
+
+    <div class="try-it">
+      <p><strong>Try it free 7 days</strong> then $20/mo. thereafter</p>
     </div>
+
+    <div class="sign-up">
+      <div class="sign-box">First Name</div>
```

Comment:
Try using inputs here. This one would be:

```
<input type="text" placeholder="First Name" />
```

Reply:
Cool, thanks for the tip!